index.js

```diff
@@ -1,6 +1,43 @@
+const yaml = require('js-yaml')
 const mergeArrayByName = require('./lib/mergeArrayByName')
 
 module.exports = (robot, _, Settings = require('./lib/settings')) => {
+  robot.on('installation', async context => {
```

Comment:
if you use `installation.added` then you don't need an if statement to check if the action was deleted.